install/image-references

```diff
@@ -18,3 +18,7 @@ spec:
     from:
       kind: DockerImage
       name: docker.io/openshift/origin-libvirt-machine-controllers:v4.0.0
+  - name: baremetal-machine-controllers
```

Comment:
until your image is built like all other release images in our payload, this will fail CI testing.

Reply:
Thanks, Derek.  I'll look into that next, then.

Reply:
Images would be mirrored to docker.io once https://github.com/openshift/release/pull/3066 is merged

Reply:
Until ART has published `baremetal-machine-controllers` to quay this PR cannot be merged.  

```
oc get istag -n ocp 4.0-art-latest:baremetal-machine-controllers
Error from server (NotFound): imagestreamtags.image.openshift.io "4.0-art-latest:baremetal-machine-controllers" not found
```

Please work with the aos-art channel